test/acceptance/features/bindAppToMultipleSbrs.feature

```diff
@@ -0,0 +1,54 @@
+@olm
+Feature: Bind a single application to multiple SBRs
+
+    As a user of Service Binding operator
+    I want to bind a single application to multiple SBRs
```

Comment:
This is not a new feature, it is either binding application to single or multiple services.